package.json

```diff
@@ -31,7 +31,8 @@
   "homepage": "https://github.com/adobe/helix-status#readme",
   "dependencies": {
     "@adobe/helix-fetch": "1.9.2",
-    "@adobe/helix-log": "4.5.3"
+    "@adobe/helix-log": "4.5.3",
+    "node-fetch": "2.6.1"
```

Comment:
Can't we just use the imports from helix-fetch?

Reply:
I thought it's not ready yet?

Reply:
a quick test yields:

```
error: test failed: 500 Cannot read property 'entries' of undefined
```

Reply:
@stefan-guggisberg it seems `@adobe/helix-fetch/FetchContext` does not export `Request` and `Response` – is there a reason not to export them?

Reply:
the reason were:
* they were not needed for the targeted use case (i.e. exposing `fetch(url, options)`)
* exporting the `Request` and `Response` classes from the underlying `fetch-h2` implementation would expose implementation specifics of `fetch-h2`; headers e.g. are guarded and only allow certain headers.
* exporting `Request` would require support for `fetch(request, options)`

for now I would stick the `node-fetch` `Request` and `Response` classes. v2 of `helix-fetch` will export general purpose `Request` and `Response` implementations.